content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -101,6 +97,8 @@ the user's email address:
     gcloud projects add-iam-policy-binding [PROJECT] --member=user:[EMAIL] --role=roles/iap.httpsResourceAccessor
     ```
 
+    Note, you need to grant the user `IAP-secured Web App User` role even if the user is already an owner or editor of the project. `IAP-secured Web App User` role is not implied by just `Project Owner` or `Project Editor` roles.
```

Comment:
Suggestion:

Correct me if I'm wrong, for better UX you could bolden **Note** here and elsewhere, if you see fit.

e.g. 

```
**Note:*** You need to grant...
```

Reply:
That's a great suggestion!